pkg/controllers/knative/deployer_controller.go

```diff
@@ -324,7 +324,8 @@ func (r *DeployerReconciler) constructConfigurationForDeployer(deployer *knative
 					Annotations: annotations,
 				},
 				Spec: servingv1.RevisionSpec{
-					PodSpec: template.Spec,
+					PodSpec:              template.Spec,
+					ContainerConcurrency: r.getConcurrency(deployer),
```

Comment:
the helper isn't doing much help

```suggestion
					ContainerConcurrency: deployer.Spec.ContainerConcurrency,
```

Reply:
"helper" removed.